database/pom.xml

```
@@ -162,17 +163,65 @@
         </pluginManagement>
       </build>
     </profile>
-    <profile>
+	<profile>
       <id>db2-111</id>
       <properties>
         <version.db2>${version.db2-11.1}</version.db2>
       </properties>
+	  <dependencies>
+	    <dependency>
+          <groupId>com.ibm.db2.jcc</groupId>
+          <artifactId>db2jcc4</artifactId>
+		  <version>${version.db2}</version>
+        </dependency>
+	  </dependencies>
+	  <build>
+        <pluginManagement>
+          <plugins>
+            <plugin>
+              <groupId>org.codehaus.mojo</groupId>
+              <artifactId>sql-maven-plugin</artifactId>
+              <dependencies>
+	            <dependency>
+                  <groupId>com.ibm.db2.jcc</groupId>
+                  <artifactId>db2jcc4</artifactId>
+                  <version>${version.db2}</version>
+                </dependency>
+              </dependencies>
+            </plugin>
+          </plugins>
+        </pluginManagement>
+      </build>
     </profile>
     <profile>
       <id>db2-105</id>
       <properties>
         <version.db2>${version.db2-10.5}</version.db2>
       </properties>
+	  <dependencies>
+	    <dependency>
+          <groupId>com.ibm.db2.jcc</groupId>
+          <artifactId>db2jcc4</artifactId>
+		  <version>${version.db2}</version>
+        </dependency>
+	  </dependencies>
+	  <build>
+        <pluginManagement>
+          <plugins>
+            <plugin>
+              <groupId>org.codehaus.mojo</groupId>
+              <artifactId>sql-maven-plugin</artifactId>
+              <dependencies>
+	            <dependency>
+                  <groupId>com.ibm.db2.jcc</groupId>
+                  <artifactId>db2jcc4</artifactId>
+                  <version>${version.db2}</version>
+                </dependency>
+              </dependencies>
+            </plugin>
+          </plugins>
+        </pluginManagement>
+      </build>
```

```suggestion
      <dependencies>
        <dependency>
          <groupId>com.ibm.db2.jcc</groupId>
          <artifactId>db2jcc4</artifactId>
        </dependency>
      </dependencies>
      <build>
        <pluginManagement>
          <plugins>
            <plugin>
              <groupId>org.codehaus.mojo</groupId>
              <artifactId>sql-maven-plugin</artifactId>
              <dependencies>
                <dependency>
                  <groupId>com.ibm.db2.jcc</groupId>
                  <artifactId>db2jcc4</artifactId>
                  <version>${version.db2}</version>
                </dependency>
              </dependencies>
            </plugin>
          </plugins>
        </pluginManagement>
      </build>
```
* The formatting looks a bit off
* We can get rid of the `version` element when defining the dependency in the `dependencyManagement` section